src/index.html

```diff
@@ -11,12 +11,101 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          class="img"
+          src="images/logo.png"
+          alt="MOYO header logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
```suggestion
          </li>
          
          <li class="nav__item">
```

<img width="1047" alt="image" src="https://github.com/user-attachments/assets/11ef67b9-fd6b-4273-bedb-b386687084a5">